### PR TITLE
Add recipe for enchantr

### PR DESCRIPTION
**Pull request by ssnn-airr**
Added recipe for the R package (not CRAN) enchantr.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by ggabernet** [comment]
@bioconda/core do you have an idea on what is wrong with this recipe? Thanks a lot for the help in advance!

**Comment by dpryan79** [comment]
`buid.sh` needs to be renamed `build.sh`

File: recipes/r-enchantr/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,98 @@
+{% set name = 'r-enchantr' %}
+{% set version = '0.0.1' %}
+
+{% set native = 'm2w64-' if win else '' %}
+{% set posix = 'm2-' if win else '' %}
```
Comment (dpryan79) [inline recipes/r-enchantr/meta.yaml:5]: Lines 3.5 can be removed, we don't build for windows.

File: recipes/r-enchantr/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+
```
Comment (dpryan79) [inline recipes/r-enchantr/meta.yaml:25]: You can remove lines 23-25

File: recipes/r-enchantr/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  summary: 'Analysis of immune repertoires.
+     QC and reports for the analysis of AIRR-seq data with Immcantation'
+  license_family: AGPL
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/AGPL-3'     
+
+extra:
+  skip-lints:
+    - in_other_channels
+  recipe-maintainers:
```
Comment (dpryan79) [inline recipes/r-enchantr/meta.yaml:97]: Please remove lines 95 and 96, since this isn't in conda-forge.

**Comment by ssnn-airr** [comment]
> 

It seems I need new prescription glasses. Thanks!

**Comment by dpryan79** [comment]
@BiocondaBot please fetch artifacts

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@BiocondaBot please merge

**Pull request merged**